src/post/BodyShort.js

```diff
@@ -13,9 +13,8 @@ module.exports = React.createClass({
     this.setState({ seeMore: true });
   },
   render() {
-    let body = striptags(remarkable.render(this.props.body));
+    let body = striptags(remarkable.render(this.props.body.replace(/&lt;/g, '<').replace(/&gt;/g, '>')));
```

Comment:
please put the regex in a getter function with a self explanatory name and then pass it to remarkable. No need to put it in a separate helper file if it's only used here.